widget_driver_annotation/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: widget_driver_annotation
 description: Defines the annotations used by widget_driver and widget_driver_generator to create code for your WidgetDrivers
-version: 0.0.2
+version: 0.0.3
```

Comment:
Is this okay? Or should I bump to `0.1.0` or `1.0.0` since this is a breaking change?

Reply:
I think we can stick with this since we assume low usage of this at the moment.